learn_evm/arithmetic-checks.md

```diff
@@ -0,0 +1,717 @@
+# A Guide on Performing Arithmetic Checks in the EVM
+
+The Ethereum Virtual Machine (EVM) distinguishes itself from traditional computer systems and virtual machines through several unique aspects.
+One notable difference is its treatment of arithmetic checks.
+While most architectures and virtual machines provide access to carry bits or an overflow flag,
+these features are absent in the EVM.
+Consequently, developers must manually incorporate these safeguards within the machine's constraints.
```

Comment:
`developers` is a bit misleading here. In practice developers who use solidity don't need to care about it